app/controllers/projects_controller.rb

```diff
@@ -114,7 +114,7 @@ def update
     @project = projects.first
     @project.update_from_params(project_params)
     error_response = @project.update
-    if error_response
+    if response_error?(error_response)
```

Comment:
This line throws an exception with the following steps:

1. Edit an existing project
2. Remove the contents of the required "name" field
3. Click the Update Project button

Full stack trace:

```
Started PUT "/projects/TESTBBB" for 127.0.0.1 at 2023-01-17 16:11:13 -0800
Processing by ProjectsController#update as HTML
  Parameters: {"authenticity_token"=>"[FILTERED]", "project"=>{"name"=>"", "institution"=>"Stanford BMIR", "contacts"=>"someone@void.com", "creator"=>[""], "homePage"=>"https://www.cnn.com", "description"=>"Blah", "ontologyUsed"=>[""]}, "commit"=>"Update Project", "id"=>"TESTBBB"}
Completed 500 Internal Server Error in 4930ms (ActiveRecord: 0.0ms | Allocations: 121820)


  
NoMethodError (undefined method `errors' for #<Faraday::Response:0x00007fb3832f08d0>):
  
app/controllers/application_controller.rb:198:in `response_success?'
app/controllers/application_controller.rb:202:in `response_error?'
app/controllers/projects_controller.rb:117:in `update'
```

I'm going to look at putting in some simple client-side validation to prevent form submission when required fields are blank. I don't want to do anything involved right now because we're just trying to get Rails 6.1 stable in production.

Reply:
I updated the PR to catch this;
This PR concerns errors displaying Submission creation/editions, Ontology creation/editions, Project creation/editions, and Subscriptions

I tested it locally, see  https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/160

Reply:
It is not related to Rails 6.1, but more of my implementation of https://github.com/ncbo/ontologies_api_ruby_client/pull/13 that breaks a lot of things in the UI (more than I planned, my bad)
In the future, I will do another iteration, where I will try to harmonize how we get validation errors from the client and how we display them.

Reply:
I don't think I expressed myself well. I fully understand this is not related to Rails 6.1, but rather the changes to the ontologies_api_ruby_client. I was speaking more generally - we've been experimenting with newer development workflows and deployment processes. The latest code in the master branch, which has a dependency on the latest version of the ontologies_api_ruby_client got inadvertently deployed to our production servers. In addition, the process that we have for reverting back to a previous tagged version of the code is broken. I've been wanting to get this pull request wrapped up and deployed so that we can tag a new release that includes Rails 6.1, among other things.